package.json

```diff
@@ -41,7 +41,7 @@
     "find-cache-dir": "^2.1.0",
     "is-wsl": "^1.1.0",
     "schema-utils": "^1.0.0",
-    "serialize-javascript": "^3.1.0",
+    "serialize-javascript": "^4.0.0",
```

Comment:
Doesn't it require node 10?

Reply:
Check it, no:
- https://github.com/yahoo/serialize-javascript/releases/tag/v4.0.0, only BigInt supporting